warehouse/bigquery/bigquery.go

```diff
@@ -543,7 +543,9 @@ func (bq *HandleT) LoadUserTables() (errorMap map[string]error) {
 			errorMap[warehouseutils.UsersTable] = status.Err()
 			return
 		}
-		defer bq.dropStagingTable(identifyLoadTable.stagingTableName)
+		if isDedupEnabled {
```

Comment:
Why do we need this extra condition for deleting the staging table.

Reply:
when isDedupEnabled is set `false`, temporary table is not created for identify table, it is just partitioned table i.e. identifies$2022-08-12. Cannot be deleted.

Reply:
Got clarified. Thanks.